scripts/windows/deps.ps1

```diff
@@ -1,3 +1,6 @@
+$FLUTTER_SDK_ZIP = "C:\development\flutter_windows_3.10.3-stable.zip"
+$FLUTTER_SDK_URL = "https://storage.googleapis.com/flutter_infra_release/releases/stable/windows/flutter_windows_3.10.3-stable.zip"
```

Comment:
we're on 3.10.6 now but otherwise I'd use this updated script for windows dev setup